build_raspOVOS.sh

```
@@ -29,197 +96,104 @@ fi
-
-echo "Installing OVOS Rust Messagebus..."
-bash /mounted-github-repo/packages/setup_rustbus.sh
-
-# Create virtual environment for ovos
+# Create and activate a virtual environment for OVOS.
 echo "Creating virtual environment..."
 mkdir -p /home/$USER/.venvs
 python3 -m venv --system-site-packages /home/$USER/.venvs/ovos
-
-# Activate the virtual environment
 source /home/$USER/.venvs/ovos/bin/activate
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)